src/components/atoms/ResultCountLabel/styles.tsx

```diff
@@ -35,13 +36,17 @@ const ResultCountWrapper = styled(Wrapper)`
   }
 `;
 
-const ResultCountP = styled(P)`
+const ResultCountP = styled(P)<ResultCountPProps>`
```

Comment:
It is not a paragraph, should be rather span I think ;)

Reply:
Good point, I've changed it :)